npm-example/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "github-packages-test",
```

Comment:
`GitHub Packages only supports scoped npm packages. Scoped packages have names with the format of @owner/name`

Probably should be `@atlassian/github-packages-test`